pkg/check/check_run.go

```diff
@@ -107,3 +110,10 @@ type Image struct {
 	Alt      string `json:"alt"`
 	Caption  string `json:"caption,omitempty"`
 }
+
+// Action is an custom event definition
+type Action struct {
+	Label 			string `json:"label"`
+	Identifier 	string `json:"identifier"`
+	Description string `json:"description"`
```

Comment:
This is where the linter is unhappy - lines 116 and 117, the type and tag (`string `json:"label"``) should be aligned.

Reply:
I needed to run `gofmt -s`, the code looks fine in my IDE. :thinking: